cmd/ttn-lw-stack/commands/as_db.go

```diff
@@ -227,6 +227,39 @@ var (
 			return nil
 		},
 	}
+	asDBUplinkRegistryCleanupCommand = &cobra.Command{
+		Use:   "purge-uplink-registry",
```

Comment:
This should be simply `purge`, and we can use it to delete other keys later on if needed. We should also keep the command description general.